app/_src/gateway/get-started/key-authentication.md

```diff
@@ -22,7 +22,7 @@ Common authentication methods include:
 
 With {{site.base_gateway}} controlling authentication, requests won't reach upstream services unless the client has successfully
 authenticated. This means upstream services process pre-authorized requests, freeing them from the 
-cost of authentication, which is a savings in compute time *and* development effort.
+cost of authentication, which is a saving in compute time *and* development effort.
```

Comment:
```suggestion
cost of authentication, which reduces computing time *and* development effort.
```
Hmm, I wonder if this could be rephrased to omit "savings" in general since that feels more "marketing-y" speak.